portfolio-project-matching-app/backend/daoUser.js

```diff
@@ -41,6 +42,28 @@ const createNewUserDoc = async (user) => {
     console.log(`Created user document with id: ${newDocRef.id}`);
 }
 
+const createNewUserDocWithId = async (user, id) => {
+    /*
+    DESCRIPTION:    creates new user document in Firebase Firestore Database
+                    based on user object provided. Similar to
+                    INSERT INTO 'users'. Note, only user document is
+                    created, not associations.
+
+    INPUT:          User object populated with data to be added in new
+                    document
+
+    RETURN:         NA
+    */
+    // build object to be added to Firebase as a user document
+    const userObj = {
+        email: user.email,
+        username: user.username,
+        introduction: user.introduction,
+    };
+    // add user document to Firebase
+    const newDocRef = await createDocWithId('users', userObj, id);
+}
```

Comment:
Changed this to use `createDocWithId` from `dao.js` instead of `addNewDocWithId` from `clientApp.ts`

Reply:
Sweet!!